test/multiverse/suites/thread/Envfile

```diff
@@ -0,0 +1,7 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+gemfile nil
```

Comment:
What is this envfile doing?

Reply:
Not much lol. So we need to test the chain an prepend instrumentation, but since this is for the thread and fiber classes, there are no gems we need to install to test this, so we pass nil so it only creates a gemfile with the base multiverse gems we need.

Reply:
Cool, that makes sense :)

Reply:
This code (equivalent to `gemfile(nil)`) is invoking the `gemfile` method defined in `multiverse/lib/multiverse/envfile.rb` with `nil` as the argument.

It looks like doing so would accomplish the same as removing this `gemfile nil` line entirely. Perhaps it should be removed.

Reply:
Previously I was operating under the assumption that a `gemfile` call was needed to get a `Gemfile.1` file created. But given this line in the `gemfile` method:

```
@gemfiles.push(content) unless content.nil? || content.empty?
```

that doesn't appear to be the case.